ui/static/ui/js/listing.js

```diff
@@ -54,5 +126,96 @@ define(['jquery', 'bootstrap', 'icheck', 'csrf'], function(jQuery) {
         event.preventDefault();
       }
     });
+
+    //open the lateral panel for members
+    $('.btn-members').on('click', function(event) {
+      event.preventDefault();
+      //remove any alert
+      $('#members-alert').empty();
+      //reset the form values
+      resetUserGroupForm();
+      //make panel visible
+      $('.cd-panel-members').addClass('is-visible');
+      //retrieve all the members
+      showUpdateAllMembers();
+      //
+    });
+    //close the lateral panel for members
+    $('.cd-panel-members').on('click', function(event) {
+      if ($(event.target).is('.cd-panel-members') ||
+          $(event.target).is('.cd-panel-close')) {
+        $('.cd-panel-members').removeClass('is-visible');
+        event.preventDefault();
+      }
+    });
+    //add button for the members
+    $(document).on('click', '.cd-panel-members-add', function() {
+      var url = $('.cd-panel-members').data('members-url');
+      var username = $('input[name=\'members-username\']').val();
+      var groupType = $('select[name=\'members-group_type\']').val();
+      // /api/v1/repositories/my-rep/members/groups/<group_type>/users/
+      url += 'groups/' + groupType + '/users/';
+      if (!isMITEmail(username)) {
+        var message = '<strong>' + username +
+          '@mit.edu</strong> does not seem to be a valid email';
+        showMembersAlert(message, 'danger');
+        return;
+      }
+      $.ajax({
+        url: url,
+        type: 'POST',
+        data: {username: username}
+      })
+      .done(function() {
+        //reset the values
+        resetUserGroupForm();
+        //show alert
+        var message = '<strong>' + username +
+          '@mit.edu</strong> added to group <strong>' +
+          formatGroupName(groupType) + '</strong>';
+        showMembersAlert(message);
+        //retrieve the members lists
+        showUpdateAllMembers();
+      })
+      .error(function(data) {
+        //show alert
+        var message = 'Error adding user ' + username +
+          '@mit.edu to group ' + formatGroupName(groupType);
+        message = message + '<br>' + data.responseJSON.username[0];
+        showMembersAlert(message, 'danger');
+      });
+    });
+    //remove button for the members
+    $(document).on('click', '.cd-panel-members-remove', function() {
+      var url = $('.cd-panel-members').data('members-url');
+      var username = $(this).data('username');
+      var groupType = $(this).data('group_type');
+      // /api/v1/repositories/my-rep/members/groups/<group_type>/users/<username>/
+      url += 'groups/' + groupType + '/users/' + username + '/';
+      $.ajax({
+        url: url,
+        type: 'DELETE'
+      })
+      .done(function() {
+        //show alert
+        var message = '<strong>' + username +
+          '@mit.edu</strong> deleted from group <strong>' +
```

Comment:
nit: The deletion might be nicer as `alert-warning` since you did do something destructive instead of green